### PR TITLE
[kueuectl] Add list resourceflavor command.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
Add `kueuectl create resourceflavor` command.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
CLI: Added list resourceflavor command.
```

### CODE REVIEW

File: pkg/util/testing/wrappers.go [hunk 1]
```diff
@@ -831,6 +831,15 @@ func (rf *ResourceFlavorWrapper) Obj() *kueue.ResourceFlavor {
 	return &rf.ResourceFlavor
 }
 
+// ObjectMetaLabel sets the label on the ResourceFlavor.
+func (rf *ResourceFlavorWrapper) ObjectMetaLabel(k, v string) *ResourceFlavorWrapper {
```
Comment (trasc) [inline pkg/util/testing/wrappers.go:835]:
```suggestion
func (rf *ResourceFlavorWrapper) Label(k, v string) *ResourceFlavorWrapper {
```
 to have similar naming for similar action as other wrappers

File: pkg/util/testing/wrappers.go [hunk 2]
```diff
@@ -831,6 +831,15 @@ func (rf *ResourceFlavorWrapper) Obj() *kueue.ResourceFlavor {
...
+	if rf.ObjectMeta.Labels == nil {
+		rf.ObjectMeta.Labels = map[string]string{}
+	}
+	rf.ObjectMeta.Labels[k] = v
+	return rf
+}
+
 // Label add a label kueue and value pair to the ResourceFlavor.
 func (rf *ResourceFlavorWrapper) Label(k, v string) *ResourceFlavorWrapper {
 	rf.Spec.NodeLabels[k] = v
```
Comment (trasc) [inline pkg/util/testing/wrappers.go:845]: Rename this to `NodeLabel` or similar.

Comment (mbobrovskyi) [reply pkg/util/testing/wrappers.go:845]: it will take a lot of changes on project. Is it ok?

Comment (mbobrovskyi) [reply pkg/util/testing/wrappers.go:845]: I just thought maybe create another cleanup PR to fix it...

File: pkg/util/testing/wrappers.go (hunk 1 above)
Comment (mbobrovskyi) [reply pkg/util/testing/wrappers.go]: Done

File: pkg/util/testing/wrappers.go (hunk 2 above)
Comment (mbobrovskyi) [reply pkg/util/testing/wrappers.go]: Done

**Review by trasc — COMMENTED** [review]
/lgtm

File: keps/2076-kueuectl/README.md [hunk 3]
```diff
@@ -225,6 +226,18 @@ Output:
...
+
+Lists ResourceFlavors. Format:
+
+```
+kueuectl list resourceflavors 
+```
+
+Output:
+
+* Name
```
Comment (alculquicondor) [inline keps/2076-kueuectl/README.md:239]: uhm... maybe it's worth adding the node labels, but it can be a separate PR to update the CRD

**Review by alculquicondor — COMMENTED** [review]
/approve

**Comment by alculquicondor** [comment]
/hold cancel
(can be done in follow up)

File: keps/2076-kueuectl/README.md (hunk 3 above)
Comment (mbobrovskyi) [reply keps/2076-kueuectl/README.md:239]: Follow up #2557
